Sources/Parser/Parser+Expressions.swift

```diff
@@ -151,6 +154,36 @@ extension Parser {
     return InoutExpression(ampersandToken: ampersandToken, expression: expression)
   }
 
+  // MARK: External Calls
+  func parseExternalCall(upTo limitTokenIndex: Int) throws -> ExternalCall {
+    try consume(.call, or: .badDeclaration(at: latestSource))
+
+    var arguments: [FunctionArgument] = []
+    if tokens[currentIndex].kind == .punctuation(.openBracket) {
+      (arguments, _) = try parseFunctionCallArgumentList()
+    }
+
+    var mode = ExternalCall.Mode.normal
```

Comment:
I would prefer 

`var mode: ExternalCall.Mode` = .normal

And then using `.bla` notation.

Reply:
fixed